CONTRIBUTING.md

```diff
@@ -1,79 +1,81 @@
 # Contributing
 
-Thank you for reaching the contribution page and showing the true gremlin in you!
-In Grommet we do believe that the more the merrier, and we are welcoming you for making this step of joining our contribution community and helping us make Grommet the best way to streamline the way you develop apps.
-You came to the right place to start your contribution, follow the guidlines and let us know if we can help with anything else.
-
-## Grommet projects
-
-Grommet is divided into a several projects; the following are notable:
-
-- [grommet](https://github.com/grommet/grommet) - the primary
-  Grommet 2.X project is actively developped and contributing is more than welcome! Be sure to check the [good first issues](https://github.com/grommet/grommet/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-- [grommet-icons](https://github.com/grommet/grommet-icons) -
-  Iconography for Grommet and React.js
-- [react-desc](https://github.com/grommet/react-desc)
-  Add a schema to your React components based on React PropTypes
-- [design-kit](https://github.com/grommet/design-kit)
-  The Grommet Design Kit provides a set of sticker sheets and templates to help bootstrap your design process.
-
-## YOU can Become a Contributor
-
-Quick steps and ideas of how you can become a contributor!
-
-1.  Code Code Code (and file Pull Request)
-2.  Create design assets or style guide revsions
-3.  Submit updates and improvements to the documentation.
-4.  Submit articles and guides which are also part of the documentation.
-5.  Join the Slack to help and interact with grommet users
-6.  Help a Grommet designer or developer by answering questions on
-    StackOverflow, Slack and GitHub.
-7.  Report bugs and proposing new features by filing issues on Github or ask in our Slack community
-8.  Share with us exciting projects using grommet on Slack
-
-## Contributing
+Thank you for reaching the contribution page and showing the true gremlin in
+you! In Grommet we do believe that the more the merrier – Welcome! Thank you for
+making this step of joining and contributing to our community, and for helping
+us make Grommet the best tool for streamlining the way you develop apps. You
+came to the right place to start your contribution! Follow the guidelines and
+let us know if we can help with anything else.
+
+## Grommet Projects
+
+Grommet is divided into several projects, the following are notable:
+
+- [grommet] – the primary Grommet 2.X project is actively developed and
```

Comment:
why do you think the link to the repo should be removed?
i.e. `(https://github.com/grommet/grommet) `

Reply:
Ahh, that’s part of the reference-style vs. the inline-style for links in Markdown. These three are equivalent:

```markdown
- [grommet](https://github.com/grommet/grommet)
- [grommet][grommet]
- [grommet]

[grommet]: https://github.com/grommet/grommet
```